src/main/java/Duke/Command/Command.java

```diff
@@ -0,0 +1,29 @@
+package Duke.Command;
+
+import Duke.TaskList;
+import Duke.Ui;
+import Duke.FileStorage.Storage;
+
+/**
+ * This abstract represents the commands provided by Duke. Currently 
+ * possible commands are ADD, DELETE, MAKR, UNMARK, LIST, EXIT.
+ */
+public abstract class Command {
+
+    /** Returns true if this command is the exit command.
+     * 
+     * @return True if this is the exit command. False otherwise.
+     */
+    public boolean isExit() {
+        return false;
+    } 
```

Comment:
Hey, is there a reason the default implementation returns `false`? Might be better to just leave it as an abstract method right?